security/remember_me.rst

```diff
@@ -176,90 +169,26 @@ visiting the site.
 
 In some cases, however, you may want to force the user to actually re-authenticate
 before accessing certain resources. For example, you might allow "remember me"
-users to see basic account information, but then require them to actually
-re-authenticate before modifying that information.
-
-The Security component provides an easy way to do this. In addition to roles
-explicitly assigned to them, users are automatically given one of the following
-roles depending on how they are authenticated:
-
-``IS_AUTHENTICATED_ANONYMOUSLY``
-    Automatically assigned to a user who is in a firewall protected part of the
-    site but who has not actually logged in. This is only possible if anonymous
-    access has been allowed.
-
-``IS_AUTHENTICATED_REMEMBERED``
-    Automatically assigned to a user who was authenticated via a remember me
-    cookie.
-
-``IS_AUTHENTICATED_FULLY``
-    Automatically assigned to a user that has provided their login details
-    during the current session.
-
-You can use these to control access beyond the explicitly assigned roles.
-
-.. note::
-
-    If you have the ``IS_AUTHENTICATED_REMEMBERED`` role, then you also
-    have the ``IS_AUTHENTICATED_ANONYMOUSLY`` role. If you have the ``IS_AUTHENTICATED_FULLY``
-    role, then you also have the other two roles. In other words, these roles
-    represent three levels of increasing "strength" of authentication.
-
-You can use these additional roles for finer grained control over access to
-parts of a site. For example, you may want your user to be able to view their
-account at ``/account`` when authenticated by cookie but to have to provide
-their login details to be able to edit the account details. You can do this
-by securing specific controller actions using these roles. The edit action
-in the controller could be secured using the service context.
-
-In the following example, the action is only allowed if the user has the
-``IS_AUTHENTICATED_FULLY`` role::
-
-    // ...
-    use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+users to change their password. You can do this by leveraing a few special "roles"::
 
+    // src/Controller/AccountController.php
     // ...
-    public function edit()
+    
+    public function accountInfo()
     {
-        $this->denyAccessUnlessGranted('IS_AUTHENTICATED_FULLY');
+        // allow any authenticated user - we don't care if they just
+        // logged in, or are logged in via a remember me cookie
+        $this->denyAccessUnlessGranted('IS_AUTHENTICATED_REMEMBERED');
 
         // ...
     }
 
-If you have installed `SensioFrameworkExtraBundle`_ in your application, you can also secure
```

Comment:
Why have you removed the mention of `SensioFrameworkExtraBundle` and it's `@Security` annotation?